.github/workflows/tests-vmtests-imagecreator.yml

```diff
@@ -119,8 +119,10 @@ jobs:
       run: |
         set -eux
 
-        # Download the test RPM and tools file.
-        ./repo/toolkit/tools/internal/testutils/testrpms/download-test-utils.sh -t 3.0 -s true
+        # Download the test RPM and tools file for azurelinux
+        ./repo/toolkit/tools/internal/testutils/testrpms/download-test-utils.sh -d azurelinux -t 3.0 -s true
+        # Download the test RPM and tools file for fedora
+        ./repo/toolkit/tools/internal/testutils/testrpms/download-test-utils.sh -d fedora -t 42 -s true
```

Comment:
The Fedora container image is currently being pulled from `registry.fedoraproject.org`. Pulling from an external registry is bad for a number of reasons including:
- The reliability of our builds becomes dependent on the uptime on the external resource,
- We are consuming someone else's resources (e.g. egress costs),
- We may end up being subject to pull limits.

We should setup an [ACR artifact cache](https://learn.microsoft.com/en-us/azure/container-registry/artifact-cache-overview) for `registry.fedoraproject.org`.

Reply:
This is a blocking issue.